io.catenax.fleet.vehicles/RELEASE_NOTES.md

```diff
@@ -0,0 +1,14 @@
+# Changelog
+All notable changes to this model will be documented in this file.
+
+## [Unreleased]
+
+## [1.0.0] - 2023-05-15
```

Comment:
not up to date

Reply:
Hi @catroest , what you mean exactly? This model has the inital version 1.0.0 and I was created 15th May

Reply:
I'm sorry for my unclear formulation. I meant the date. As far as I know it should be the merge/MS3 date